views/patient/new.html

```diff
@@ -53,7 +54,7 @@
                 gender: null, birthdate: null, birthdate_estimated: null,
                 home_district: null, home_traditional_authority: null, home_village: null,
                 current_district: null, current_traditional_authority: null, current_village: null,
-                landmark: null, cell_phone_number: null, occupation: null, relationship: null,
+                landmark: null,malawi_national_id: null, cell_phone_number: null, occupation: null, relationship: null,
```

Comment:
Could the showing of the question for national ID be optional? Like the way we do with military things. They should only apply to the OPD program for now. Also in testing right now I've seen when I specify unknown the summary shows the national ID as unidentified.

Reply:
Alright i am working on it

Reply:
i have worked on it, now malawi national id is optional and only apply to OPD